config.example.js

```diff
@@ -61,7 +61,6 @@ export const aptly = {
 export const database = 'mongodb://localhost/houston-dev'
 
 export const server = {
-  port: 3000,
```

Comment:
Why are you removing the port here, this port configuration should stay seperate from the public server url (where you put it) as we want to route through nginx usually.